### PR TITLE
Release/1.9.0

### PR DESCRIPTION
**Pull request by pimjansen**
* 1.9.0 changelog updated
* Skipped test that download the actual image. This should be part of an integration test and not an unittest.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -458,8 +557,7 @@ CHANGELOG
 * Added English for New Zealand (en\_NZ) locale [\#283](https://github.com/fzaninotto/Faker/pull/283) ([JasonMortonNZ](https://github.com/JasonMortonNZ))
 * Added mention of external Provider for cron expressions to readme[\#498](https://github.com/fzaninotto/Faker/pull/498) ([swekaj](https://github.com/swekaj))
 
-2014-06-04, v1.4.0
-------------------
+##2014-06-04, v1.4.0
```
Comment (localheinz) [inline CHANGELOG.md]:
```suggestion
## 2014-06-04, v1.4.0
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -347,8 +447,7 @@ CHANGELOG
 - Updated `optional` and `boolean` apis to be more consistent [\#513](https://github.com/fzaninotto/Faker/pull/513) ([EmanueleMinotto](https://github.com/EmanueleMinotto))
 - Added Greek (el\_GR) `Address` provider [\#504](https://github.com/fzaninotto/Faker/pull/504) ([drakakisgeo](https://github.com/drakakisgeo))
 
-2015-05-29, v1.5.0
-------------------
+##2015-05-29, v1.5.0
```
Comment (localheinz) [inline CHANGELOG.md]:
```suggestion
## 2015-05-29, v1.5.0
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,8 +1,110 @@
-CHANGELOG
-=========
+# CHANGELOG
 
-2018-07-12, v1.8.0
-------------------
+
+##2019-11-10, v1.9.0
```
Comment (localheinz) [inline CHANGELOG.md]:
```suggestion
## 2019-11-10, v1.9.0
```

**Comment by pimjansen** [comment]
@localheinz comments resolved

**Pull request merged**